app/templates/components/login-form.emblem

```diff
@@ -0,0 +1,49 @@
+if errorMessage
+  .alert.alert-danger.alert-dismissible role="alert"
+    button.close type="button" data-dismiss="alert" aria-label="Close"
+      span hidden="true" &times;
+    = errorMessage
+
+form submit="authenticateWithDjango"
+  .row
+    .col-xs-12
+      .form-group class=emailError:has-error:
+        .input-group
+          span.input-group-addon
+           .icon
+              fa-icon "envelope"
+          = input class='form-control' value=identification placeholder='Your Email Address'
+  .row
+    .col-xs-12
+      .form-group class=passwordError:has-error
+        .input-group
+          span.input-group-addon
+           .icon
+              fa-icon "key"
+          = input class='form-control' value=password placeholder='Your Password' type='password'
+  .row.login-links
+    .col-xs-3.col-sm-2.col-md-1
```

Comment:
I'd make the buttons equally large and take the whole width on a mobile screen for consistency and and fat fingers :wink: . `col-xs-3 col-sm-3 col-md-2` and changing the button width to 100%.